### PR TITLE
fix(Autosuggest): Increase suggestion item height

### PR DESCRIPTION
**Pull request by kosanna**
Increased suggestion height to 5 rows as per https://github.com/SEEK-Jobs/chalice/issues/1071
Cleanup tests.

### CODE REVIEW

File: react/Autosuggest/Autosuggest.test.js [hunk 1]
```diff
@@ -32,70 +20,54 @@ const getAutosuggestProps = (suggestions = []) => ({
...
-
-    sinon.stub(console, 'error', errorMessage => {
-      errors.push(errorMessage);
-    });
+    global.console = { error: jest.fn() };
   });
 
   afterEach(() => {
-    console.error.restore();
+    jest.clearAllMocks();
```
Comment (markdalgleish) [inline react/Autosuggest/Autosuggest.test.js]: This is no longer restoring the native console error method.

Comment (kosanna) [inline react/Autosuggest/Autosuggest.test.js]: The overwrite only effects current test, so all good.

Comment (markdalgleish) [inline react/Autosuggest/Autosuggest.test.js]: It's still clobbering `global.console`, though. Jest won't be able to reverse this.

Comment (kosanna) [inline react/Autosuggest/Autosuggest.test.js]: Agree, fixed

**Comment by markdalgleish** [comment]
@kosanna just a heads up — I renamed this PR, in case you missed it.

**Comment by kosanna** [comment]
Anyone, second approval @here?

**Comment by markdalgleish** [comment]
@kosanna I think your at-here just pinged some poor random guy on GitHub 😅

**Pull request merged**